docking/dock_and_equilibrate.py

```diff
@@ -153,7 +153,7 @@ def dock_and_equilibrate(
         # collect a du_dl calculation once every other step
         subsample_interval = 1
 
-        full_du_dls, _ = ctxt.multiple_steps(
+        full_du_dls, _, host_box = ctxt.multiple_steps(
```

Comment:
@schmolly I used the box here as the report_step call relies on the box coords. This might need more detailed review to ensure this behaves as you expect it too. Same with the docking/pose_dock.py change

Reply:
I undid this as the desired behavior was to return the boxes for each x interval, which would give you a different box.